<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>参数</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>E:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditE"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>t:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditT"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>v:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditV"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>lc值：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lcValue"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>等效应力范围</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="colorBar">
            <property name="lineWidth">
             <number>11</number>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_10">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLineEdit" name="lineEditMin"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditMax"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>信息获取</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>位置坐标</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>x:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPointX"/>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPointY"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>应力信息</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>σ_x：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sigmaX"/>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>σ_y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sigmaY"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>τ</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tau"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>应变信息</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>ε_x:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="epsilonX"/>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>ε_y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="epsilonY"/>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>γ:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="gamma"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>位移信息</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>u:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mu"/>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>v:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nu"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
